src/common/wizard/index.css

```diff
@@ -16,7 +16,7 @@
   flex: 0 0 15em;
 }
 .title.success {
-  color: #209020;
+  color: #089944;
```

Comment:
When you need to use multiple times the same value, use [CSS values](https://github.com/tivac/modular-css#values).